manifest.json

```diff
@@ -7,6 +7,32 @@
               "48": "icons/icon48.png",
              "128": "icons/icon128.png" },
   "minimum_chrome_version": "69.0",
+  "commands": {
+    "jump-back-tab": {
+      "description": "Jump back in the tab stack."
```

Comment:
Everywhere else you call it a tab list (instead of a stack). These should be consistent.